cocos/core/renderer/scene/submodel.ts

```diff
@@ -41,6 +42,27 @@ export class SubModel {
     protected _priority: RenderPriority = RenderPriority.DEFAULT;
     protected _inputAssembler: InputAssembler | null = null;
     protected _descriptorSet: DescriptorSet | null = null;
+    protected _wireframeMode: WireframeMode = WireframeMode.SHADED;
+    protected _wireframeIa: InputAssembler | null = null;
+
+    setWireframe (val: WireframeMode, buffer: Buffer|null) {
```

Comment:
As i think, just need to set Model's rendering mode, then Model can generate needed data.

Reply:
But the purpose of the data is to generate wireframeIa. If the rendering mode of the model does not need to generate wireframeIa, then wireframeia is null. Not conflicting

Reply:
Yep, i see. If it is not needed, then rendering mode is normal. I do not mean it can not work currently, just mean that the API is not elegant.

Reply:
If the submodel stores the original data(uint16array), I think it would be better to generate wireframe inputassempler. Since the InputAssempler attribute is already in the Submodel, here just add wireframeia to make the distinguish

Reply:
Ok, please add comment to describe the hack.